types/lodash/index.d.ts

```diff
@@ -7221,296 +7257,879 @@ declare namespace _ {
         ): T|undefined;
     }
 
-    interface LoDashExplicitWrapperBase<T, TWrapper> {
+    interface LoDashExplicitWrapperBase<T, TWrapper> {
+        /**
+         * @see _.findLast
+         */
+        findLast(
+            predicate?: ListIterator<T, boolean>,
+            fromIndex?: number
+        ): any;
+
+        /**
+         * @see _.findLast
+         */
+        findLast(
+            predicate?: string,
+            fromIndex?: number
+        ): any;
+
+        /**
+         * @see _.findLast
+         */
+        findLast(
+            predicate?: PartialObject<T>,
+            fromIndex?: number
+        ): any;
+    }
+
+    //_.flatMap
+    interface LoDashStatic {
+        /**
+         * Creates an array of flattened values by running each element in collection through iteratee
+         * and concating its result to the other mapped values. The iteratee is invoked with three arguments:
+         * (value, index|key, collection).
+         *
+         * @param collection The collection to iterate over.
+         * @param iteratee The function invoked per iteration.
+         * @return Returns the new flattened array.
+         */
+        flatMap<T, TResult>(
+            collection: List<T> | null | undefined,
+            iteratee?: ListIterator<T, Many<TResult>>
```

Comment:
@andy-ms then `_.identity` will be used as the iteratee, so you basically just flatten the collection.

There are a number of issues with the `flatMap` overloads (as you've noticed) which are addressed as part of my other PR (https://github.com/DefinitelyTyped/DefinitelyTyped/pull/18663)

That's going to make a bit of a mess due to merge conflicts... I'm thinking once my PR is merged, I'll make a new PR based on @e-cloud 's fork, and update `flatMapDeep`/`flatMapDepth` to include the changes I made to `flatMap`.